### PR TITLE
Solve ensemble conflicts

### PR DESCRIPTION
**Pull request by aptsunny**
*Issue #, if available:*

*Description of changes:*
Solve ensemble conflicts

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-202-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-202/1/index.html

File: examples/image_classification/README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
 # Kaggle Benchmark by Autogluon 
 | Datset | GluonCV Baseline/Autogluon/1st/Rank | Search Space(net.choice/learning_rate/momentum/wd) | Configuration(epochs/trials/batchsize/gpus_per_trial) | Training Log/Training Command |
 |:-------:|:-----:|:-------:|:-------:|:-------:|
+<<<<<<< HEAD
```
Comment (zhanghang1989) [inline examples/image_classification/README.md]: merging conflict

File: examples/image_classification/benchmark.py [hunk 2]
```diff
@@ -43,7 +43,11 @@ def main():
         os.mkdir(opt.dataset)
     local_path = os.path.dirname(__file__)
     data_path = os.path.join(local_path, opt.data_dir, opt.dataset)
+<<<<<<< HEAD
```
Comment (zhanghang1989) [inline examples/image_classification/benchmark.py]: please resolve conflicts

File: examples/image_classification/kaggle_configuration.py [hunk 3]
```diff
@@ -271,7 +271,11 @@ class NAG(optim.NAG):
             learning_rate=ag.space.Real(1e-4, 1e-2, log=True),
             momentum=ag.space.Real(0.85, 0.95),
             wd=ag.space.Real(1e-6, 1e-2, log=True),
+<<<<<<< HEAD
```
Comment (zhanghang1989) [inline examples/image_classification/kaggle_configuration.py]: same here

File: tests/unittests/test_classification_tricks.py [hunk 4]
```diff
@@ -89,7 +89,10 @@ def test_tricks(test_trials):
                               num_trials = target['num_trials'],
                               batch_size = target['batch_size'],
                               verbose = True,
+<<<<<<< HEAD
```
Comment (zhanghang1989) [inline tests/unittests/test_classification_tricks.py]: conflict

**Comment by Innixma** [comment]
Can we please make a point to rebase instead of merge in development branches. Merging is dangerous when not done correctly. We should not be committing code with merge conflicts, even into development branches.

**Comment by zhanghang1989** [comment]
> Can we please make a point to rebase instead of merge in development branches. Merging is dangerous when not done correctly. We should not be committing code with merge conflicts, even into development branches.

Agree. Merging the code should come after the CI finishes.

I think the confilicts in the code is a mistake.

**Comment by Innixma** [comment]
> I think the confilicts in the code is a mistake.

I understand it's a mistake, I'm saying that developers should be keenly aware of what they are committing so that those mistakes are not made in the first place. On larger commits this can become a major issue.

**Comment by szha** [comment]
Job PR-202-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-202/2/index.html

**Comment by zhanghang1989** [comment]
> > I think the confilicts in the code is a mistake.
> 
> I understand it's a mistake, I'm saying that developers should be keenly aware of what they are committing so that those mistakes are not made in the first place. On larger commits this can become a major issue.
> 
> By rebasing instead of merging, it is much harder for these mistakes to happen.

I agree. FYI @aptsunny , let's wait the CI build finishes for the last commit before merging PRs.

**Pull request merged**